saleor/extensions/plugins/avatax/plugin.py

```diff
@@ -154,7 +154,7 @@ def calculate_checkout_total(
         total_gross = Money(amount=total_net + tax, currency=currency)
         total_net = Money(amount=total_net, currency=currency)
         total = TaxedMoney(net=total_net, gross=total_gross)
-        voucher_amount = checkout.discount_amount
+        voucher_amount = checkout.discount
```

Comment:
`checkout.discount` is `Money` instance; I think the variable name `voucher_amount` will be **very** confusing in the future.
Maybe it should be `voucher_value` or something else?